NuRadioReco/utilities/trace_utilities.py

```diff
@@ -15,92 +15,49 @@
 # see Phys. Rev. D DOI: 10.1103/PhysRevD.93.122005
 # to convert V**2/m**2 * s -> J/m**2 -> eV/m**2
 
+def get_efield_antenna_factor(*args, **kwargs):
+    logger.warning("get_efield_antenna_factor is moved to NuRadioReco.utilities.signal_processing.get_efield_antenna_factor")
+    return signal_processing.get_efield_antenna_factor(*args, **kwargs)
```

Comment:
Can you change this to a DeprecationWarning? (One day I _will_ try to make the CI pick up on these, promise!)

Reply:
I obeyed but I am not so happy with that since these warnings are not nicely integrated in our logging system - however that should be possible n0? https://docs.python.org/3/library/logging.html#logging.captureWarnings